Two-body-scf-guess.py

```diff
@@ -0,0 +1,112 @@
+from wrap_pyscf import *
+import numpy as np
+
+def make_monomer_dens(monomers, bs):
+    monomer_dens = []
+    for j in monomers:
+        k = density(j, bs)
+        monomer_dens.append(k)
+    return monomer_dens
+monomer_dens = make_monomer_dens(monomers, bs)
```

Comment:
When you do `python3 Two-body-scf-guess.py` this line will call the function `make_monomer_dens` with the arguments `monomers` and `bs` (which as far as I can tell are not defined so this should raise an error, I'm actually not sure how this script works...) and assign the result to the variable `monomer_dens`. Ideally the file should be structured like:

```.py
# Your imports
# You function definitions

if '__name__` == `__main__` :
    # Definition of your system
    # Call the functions with your system
```